### PR TITLE
feat(pci.data-integration): datatr-286 - add data-integration project

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-286

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-286
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add data-integration project in beta

### :flags: Translations
 
- [x] TRANS-52526

### CODE REVIEW

**Review by Dimfacion — APPROVED** [review]
Nice :)

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/cli/cli.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<h3>Command Line Interface (CLI)</h3>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-integration/tabs/cli/cli.html:1]: **question:** Dont we need trans for this?

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/style.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pci-project-data-integration-jobs-list {
+  .oui-pagination {
+    margin-top: 10px;
+  }
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/style.scss:4]: Can you please avoid overriding the styles specific for few screens? It doesn't provide the same user experience to the users across the Manager.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/style.scss (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/style.scss:4]: Hi @anooparveti , this was requested, as the `oui-tile` component containing the table seems to overwrite default margin to 0, which does not look good.
image

I have checked some datagrids that are on the manager, and they do have a margin on top of pagination: 
image

I think it makes sense to include a margin for this specific table where a margin above the pagination is requested. 
However, maybe it should be handled in the `ovh-ui-kit`. I can declare a bug in the project, and once it is solved, we would be able to remove this custom css here.

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/jobs-list.component.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import template from './jobs-list.html';
+import controller from './jobs-list.controller';
+
+import './style.scss';
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/jobs-list.component.js:4]:
```suggestion
import controller from './jobs-list.controller';
import './style.scss';
```

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  /* @ngInject */
+  constructor($scope) {
+    this.$scope = $scope;
+  }
+
+  $onInit() {
+    this.WORKFLOW_STATUSES = WORKFLOW_STATUSES;
+    this.columns = WORKFLOW_TABLE_COLUMNS;
+    this.itemsPerPage = ITEMS_PER_PAGE;
+    this.paginationPage = 1;
```
Comment (sachinrameshn) [inline packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js:18]: Suggestion: Can you fetch 1 from constant file?

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/jobs-list.component.js (hunk 3 above)
Comment (abullet33) [reply packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/jobsList/jobs-list.component.js]: Fixe

**Review by Dimfacion — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js:1]: **question:** Any reason why the existing `data-grid` from ui-kit cannot be used?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/data-integration/tabs/dashboard/components/workflowsList/workflows-list.controller.js:1]: Hi @anooparveti ,
The reason we cannot use the existing data-grid from the UI kit is that it doesn't support the ability for users to select a unique row, which is needed here. When a user clicks a row in the workflows data-grid, its jobs are loaded, and displayed in a second table below:

image

This design, and the look of the custom data-grid have been discussed and validated by CX team before the developments :wink:

**Review by anooparveti — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**